app/src/js/components/button.jsx

```diff
@@ -1,21 +1,42 @@
 'use strict';
 
 const { React, Base } = require('./base');
+const ColorPicker = require('react-color');
+const sweetAlert = require('sweetalert');
+
+module.exports = class Component extends Base {
+
+  constructor() {
```

Comment:
since there are no props being passed in when you render this component is `app.jsx` it isn't a problem now, but you should use `constructor(props)` here and then...